network/datadog_checks/network/network.py

```diff
@@ -472,7 +472,10 @@ def _add_conntrack_stats_metrics(self, conntrack_path, tags):
         Add the parsed metrics
         """
         try:
-            output, _, _ = get_subprocess_output(["sudo", conntrack_path, "-S"], self.log)
+            if os.environ.get('DOCKER_DD_AGENT'):
```

Comment:
I'd feel more confortable with this change if it was behind a config option

Reply:
Updated